README.md

```diff
@@ -23,7 +23,7 @@ Azure:
 [![Azure builds](https://prow.ci.openshift.org/badge.svg?jobs=periodic-ci-openshift-oadp-operator-master-4.13-operator-e2e-azure-periodic-slack)](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-oadp-operator-master-4.13-operator-e2e-azure-periodic-slack)
 </div>
 
-Note: Official Overview and usage documents are being moved to [Red Hat Customer Portal](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/backup_and_restore/index)
+Note: Official Overview and usage documents are being moved to [Red Hat Customer Portal](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/backup_and_restore/oadp-application-backup-and-restore)
```

Comment:
If it doesn't have to link to access.RH, you could use docs.openshift URL with `latest` and it'll redirect to whatever is the latest release -- in this case, 4.14.  Then it'll follow along with the OCP releases as they occur and we won't need to revisit this from time to time.

https://docs.openshift.com/container-platform/latest/backup_and_restore/application_backup_and_restore/oadp-intro.html

Reply:
thanks all!